FeedSX/screens/createPost/index.tsx

```diff
@@ -0,0 +1,549 @@
+              />
+            ) : (
+              <>
+                {/* single image selected section */}
+                {formattedMediaAttachments[0]?.attachmentMeta.format ===
+                  'image/jpg' && (
+                  <LMImage
+                    imageUrl={`${formattedMediaAttachments[0]?.attachmentMeta.url}`}
+                    showCancel
+                    onCancel={() => removeSingleAttachment()}
+                  />
+                )}
+                {/* single video selected section  */}
+                {formattedMediaAttachments[0]?.attachmentMeta.format ===
+                  'video/mp4' && (
+                  <LMVideo
+                    videoUrl={`${formattedMediaAttachments[0]?.attachmentMeta.url}`}
+                    showCancel
+                    onCancel={() => removeSingleAttachment()}
+                  />
+                )}
+              </>
+            )
+          ) : null}
+          {/* selected document view section */}
+          {formattedDocAttachments && formattedDocAttachments.length >= 1 && (
+            <LMDocument
+              attachments={formattedDocAttachments}
+              showCancel
+              onCancel={index => removeDocAttachment(index)}
+            />
+          )}
+          {/* added link preview section */}
+          {formattedMediaAttachments.length <= 0 &&
+            formattedDocAttachments.length <= 0 &&
+            formattedLinkAttachments &&
+            formattedLinkAttachments.length >= 1 && (
+              <LMLinkPreview
+                attachments={formattedLinkAttachments}
+                showCancel
+                onCancel={() => {
+                  setFormattedLinkAttachments([]);
+                }}
+              />
+            )}
+        </View>
+        {/* add more media button section */}
+        {(formattedMediaAttachments.length > 0 ||
+          formattedDocAttachments?.length > 0) && (
+          <LMButton
+            onTap={
+              formattedMediaAttachments.length > 0
+                ? () => handleGallery('mixed')
+                : formattedDocAttachments.length > 0
+                ? () => handleDoc()
+                : () => {}
+            }
+            icon={{
+              assetPath: require('../../assets/images/plusAdd_icon3x.png'),
+              type: 'png',
+              height: 20,
+              width: 20,
+            }}
+            text={{
+              text: 'Add More',
+              textStyle: {
+                fontSize: 14,
+                fontWeight: '500',
+                color: '#5046E5',
+                marginLeft: 5,
+              },
+            }}
+            buttonStyle={{
+              width: '35%',
+              borderColor: '#5046E5',
+              borderWidth: 1,
+              borderRadius: 8,
+              alignSelf: 'center',
+              paddingVertical: 8,
+              marginTop: 20,
+            }}
```

Comment:
these styles making code readability, can you put it in a separate file

Reply:
resolved